install/linux.md

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_label: Linux
+---
+
+# Installing RGBDS on Linux
+
+## Using your package manager
+
+First, please check if RGBDS is available in your distribution's software center or package manager repositories.
+If a satisfactory version is available, it is a good idea to install from there.
+
+## Using our pre-built binaries
+
+:::info Availability
```

Comment:
This info box only adds noise and it should be removed. We're mentioning anyway the build from source way, so we can just add there "e.g. if you need an older version". Here it doesn't serve for any purpose.